src/libraries/Common/src/System/Net/Http/Http2/Hpack/IntegerEncoder.cs

```diff
@@ -8,6 +8,11 @@ namespace System.Net.Http.HPack
 {
     internal static class IntegerEncoder
     {
+        /// <summary>
```

Comment:
Move IntegerEncoder out of Http2 (as it's used by Http3)?

Reply:
Need to figure out the sharing script for this. Maybe just merge the http/2 and http/3 into a single shared directory structure.

Reply:
Yep, that's in line with what I was thinking.